src/content/docs/en/guides/rss.mdx

```diff
@@ -154,7 +154,7 @@ items: import.meta.glob('./blog/*.{md,mdx}'),
 The `content` key contains the full content of the post as HTML. This allows you to make your entire post content available to RSS feed readers.
 
 :::tip
-A package like [`sanitize-html`](https://www.npmjs.com/package/sanitize-html) will make sure that your content is properly sanitized, escaped, and encoded.
+A package like [`sanitize-html`](https://www.npmjs.com/package/sanitize-html) will make sure that your content is properly sanitized, escaped, and encoded. In the process, such a package might also remove some harmless elements and attributes, so make sure to verify the output and configure the package according to your needs.
```

Comment:
Could be useful to link to something like https://github.com/apostrophecms/sanitize-html#common-use-cases, but I didn't want to make this tip too verbose

Reply:
@mayank99 Although not a direct link to the anchor, the npm readme does include the "common use cases" so someone can certainly get there just by reading further down the page.

Let's merge the tip as is for now, which I think sounds great!